android/app/src/main/java/app/organicmaps/sound/MediaPlayerWrapper.java

```diff
@@ -51,11 +51,13 @@ public void release()
   }
 
   @NonNull
+  @SuppressWarnings("deprecation") // https://github.com/organicmaps/organicmaps/issues/3632
   private static AsyncTask<Integer, Void, InitializationResult> makeInitTask(@NonNull MediaPlayerWrapper wrapper)
   {
     return new InitPlayerTask(wrapper);
   }
 
+  @SuppressWarnings("deprecation") // https://github.com/organicmaps/organicmaps/issues/3632
```

Comment:
How playback method is related to the AsyncTask deprecation warning?

Reply:
It calls asynctask.execute() inside.